### PR TITLE
Test coverage increase 

### PR DESCRIPTION
**Pull request by melvil-donnart**

### CODE REVIEW

File: lox_services/utils/Files/dummy.csv [hunk 1]
```diff
@@ -0,0 +1 @@
+this,is,a,test
```
Comment (mathisbrrt) [inline lox_services/utils/Files/dummy.csv:1]: this file can be move in the same folder than other test files

File: tests/persistence/database/test_contants.py
Comment (mathisbrrt) [inline tests/persistence/database/test_contants.py:1]: typo in file name

File: tests/proxy/test_brightdata.py
Comment (mathisbrrt) [inline tests/proxy/test_brightdata.py:1]: why everything is commented ?

File: tests/utils/Files/test_image.png
Comment (mathisbrrt) [inline tests/utils/Files/test_image.png:1]: I would put all the assets files in the same folder as we do it in main-algorithms

**Review by mathisbrrt — CHANGES_REQUESTED** [review]

File: tests/translation/test_use_locale.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    def test_get_all_locales(self):
+        # Call the function
+        result = get_all_locales()
+
+        # Assertions
+        self.assertIsInstance(result, dict)
+        self.assertTrue(len(result) > 0)
+
+    # TODO: Need permission to download locale, Github Action is not working because of that.
+    # def test_download_locale(self):
```
[the remark this answers is not in the document]
Comment (melvil-donnart) [reply tests/translation/test_use_locale.py:29]: Well, I comment it as the test runs locally but not on GitHub. Commenting it is the only way we can keep in a test file without making the tests crash

**Review by mathisbrrt — APPROVED** [review]

**Pull request merged**